roles/apache/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: install apache package
+  apt: pkg=apache2
+
+- name: install apache module packages
+  apt: pkg={{ item }}
+  with_items: apache.modules
+
+- name: enable apache modules
+  apache2_module: name={{ item }}
+  with_items:
+    - headers
+    - ssl
+    - rewrite
+    - uwsgi
+  notify: reload apache
+
+- name: disable default vhost
+  file: dest=/etc/apache2/sites-enabled/{{ item }}
+        state=absent
+  with_items:
+    - 000-default
+    - 000-default.conf
+    - default
+    - default.conf
+  notify: stop apache
```

Comment:
Why doesn't this use the apache2_site module you've added?

Also, why is there a straight stop here, instead of a restart?